ember-headlessui/addon/components/listbox.js

```diff
@@ -181,15 +187,27 @@ export default class ListboxComponent extends Component {
     optionElement.setAttribute('data-index', this.optionElements.length - 1);
```

Comment:
any chance you'd be willing to do a PR that removes all this "registerElement" stuff? it's very side-effect heavy and causes double (or more!) renders